gulpfile.js

```diff
@@ -19,21 +23,37 @@ gulp.task('serve', ['nodemon', 'sass', 'pug'], function() {
 });
 
 gulp.task('build', ['sass', 'pug']);
-
 gulp.task('pug', function () {
   return gulp.src(PATHS.pug)
     .pipe(pug())
     .pipe(gulp.dest('dist'))
     .pipe(browserSync.stream());
 });
-
 gulp.task('sass', function() {
   return gulp.src(PATHS.sass)
     .pipe(sass({outputStyle: 'compressed'}))
     .pipe(gulp.dest('dist'))
     .pipe(browserSync.stream());
 });
 
+gulp.task('lint', ['eslint', 'puglint', 'sasslint']);
+gulp.task('eslint', function(){
+  return gulp.src(PATHS.eslint)
+    .pipe(eslint())
+    .pipe(eslint.format())
+    .pipe(eslint.failAfterError());
+});
+gulp.task('puglint', function(){
+  return gulp.src(PATHS.pug)
+    .pipe(puglint());
+});
+gulp.task('sasslint', function(){
```

Comment:
:cry: we prob should use another preset :stuck_out_tongue: this one is really not cool